docs/server-config.md

```diff
@@ -95,6 +95,10 @@ at [`logging.config` documentation page](https://docs.python.org/3.6/library/log
   through the `/query` method, or using the `tabpy.query(...)` syntax with
   the `/evaluate` method.
 - `TABPY_GZIP_ENABLE` - Enable Gzip support for requests. Enabled by default.
+- `TABPY_ARROW_ENABLE` - Enable Arrow connection for data streaming.
```

Comment:
Can we add a note for the default `TABPY_ARROW_ENABLE` setting. And for the port as well.